Algorithm-Caesars-Cipher.md

```diff
@@ -1,29 +1,27 @@
-:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program. :busts_in_silhouette: Write your own code. :memo:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](http://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-to-get-help-when-you-get-stuck) if you get stuck. Try to pair program :busts_in_silhouette: . Write your own code :memo: .
 
 # :checkered_flag: Problem Explanation:
-> - You need to write a function, which will take string encoded with
+- You need to write a function, which will take string encoded with
```

Comment:
This change can be done in another PR. This PR is dedicated to markdown formatting changes in the template file and it's related files only.